### PR TITLE
Add annotation for accent omission

### LINKED ISSUE #14: wrong accent marks
Accent marks are dropped in some sentences, but they are not annotated in lemma.  In the word 19, wrong lemma is given (probably it is wrongly predicted as a plural form of *`adema`).
```
# sent_id = es-dev-003-s429
# text = Este organismo maneja mas de cinco mil empresas privadas, publicas y extranjeras en el territorio chino, ademas de que realiza estudios de mercados, deteccion de nichos de oportunidades y todo lo relacionado con productos carnicos, agricolas y pesqueros.
4	mas	mas	ADV	_	_	6	advmod	_	_
11	publicas	publico	ADJ	_	Gender=Fem|Number=Plur	9	conj	_	_
19	ademas	adema	ADV	_	_	22	advmod	_	_
```

They should have correct lemmata with `CorrectForm` annotation in MISC field.
```
4	mas	más	ADV	_	_	6	advmod	_	CorrectForm=más
11	publicas	público	ADJ	_	Gender=Fem|Number=Plur	9	conj	_	CorrectForm=públicas
19	ademas	además	ADV	_	_	22	advmod	_	CorrectForm=además
```

This is an example of wrong accent mark, which causes a wrong lemma and empty features.
```
# sent_id = es-dev-003-s450
# text = En una taberna escribío lo encontrado y luego huyó al desierto sabiendo que las tropas del Califa le buscaban.
4	escribío	escribío	VERB	_	VerbForm=Fin	0	root	_	_
```
This should be annotated as:
```
4	escribío	escribir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	_	CorrectForm=escribió
```

### PR DESCRIPTION
**Pull request by kanayamah**
Added `CorrectForm` in MISC field, and fixed lemma and features.
Closes #14 .

### CODE REVIEW

**Comment by kanayamah** [comment]
Let me update them again.  I fixed format errors:
* Order of `Number` and `NumType` features (I learned that they must be sorted alphabetically by `.lower()`)
* Remove `Typo=Yes` in MWT
* Remove whitespace in lemma

**Comment by dan-zeman** [comment]
Interesting. It did not occur to me before that Typo=Yes would make sense on the multiword-token line when it does not occur on the member word line. Maybe the guidelines should allow it... but it would be ugly to allow one feature and not the others...

**Review by dan-zeman — APPROVED** [review]
Looks good to me (but once again, I did not carefully examine every modified line, there are too many of them). The only question I have is whether _sólo_ as an adverb and spelled _solo_ should be regarded a typo.

**Comment by kanayamah** [comment]
@dan-zeman  Excuse me!  I missed your feedback comments.

> It did not occur to me before that Typo=Yes would make sense on the multiword-token line
Yes, I removed `Typo=Yes` from MWT since it blocked the validator.  Which is better, (1) allowing `CorrectForm` in MISC without `Typo=Yes` or (2) allowing only `Typo=Yes` feature (and others if needed) for MWT?

File: es_gsd-ud-test.conllu [hunk 1]
```diff
@@ -1612,7 +1612,7 @@
 # sent_id = es-dev-003-s472
 # text = Tan solo un jugador de los Bears ha llegado a jugar en la NBA, y sólamente lo hizo en 18 partidos, en 1990.
 1	Tan	tanto	ADV	_	_	2	advmod	_	_
-2	solo	solo	ADV	_	_	9	advmod	_	_
```
[the remark this answers is not in the document]
Comment (kanayamah) [reply es_gsd-ud-test.conllu:1615]: @dan-zeman I agree.  As `solo` without accent is acceptable, I'll remove `Typo` and `CorrectForm` in these cases.

[the remark this answers is not in the document]
Comment (kanayamah) [reply es_gsd-ud-test.conllu:1615]: fixed this.

**Comment by dan-zeman** [comment]
> @dan-zeman Excuse me! I missed your feedback comments.
> 
> > It did not occur to me before that Typo=Yes would make sense on the multiword-token line
> Yes, I removed `Typo=Yes` from MWT since it blocked the validator.  Which is better, (1) allowing `CorrectForm` in MISC without `Typo=Yes` or (2) allowing only `Typo=Yes` feature (and others if needed) for MWT?

Right now it is not possible to use `Typo=Yes` at the MWT line but it is possible to use anything in MISC there. If the guidelines are changed to allow `Typo=Yes` on the MWT line, then the validator will allow it too.

**Comment by kanayamah** [comment]
@dan-zeman got it.  Thank you for merging this!

**Comment by dan-zeman** [comment]
> > @dan-zeman Excuse me! I missed your feedback comments.
> > > It did not occur to me before that Typo=Yes would make sense on the multiword-token line
> > > Yes, I removed `Typo=Yes` from MWT since it blocked the validator.  Which is better, (1) allowing `CorrectForm` in MISC without `Typo=Yes` or (2) allowing only `Typo=Yes` feature (and others if needed) for MWT?
> 
> Right now it is not possible to use `Typo=Yes` at the MWT line but it is possible to use anything in MISC there. If the guidelines are changed to allow `Typo=Yes` on the MWT line, then the validator will allow it too.

It is now possible to use `Typo=Yes` on the MWT line (see https://github.com/UniversalDependencies/docs/issues/924) and the validator has been updated to respect the new policy.

**Pull request merged**